translations/case+intl-icu.da.xlf

```diff
@@ -1609,6 +1609,10 @@
         <source>Inspection</source>
         <target state="translated">Besigtigelse</target>
       </trans-unit>
+      <trans-unit id="z57ogPt" resname="Reference number">
+        <source>Reference number</source>
+        <target state="translated">Reference nummer</target>
```

Comment:
This should be “Referencenummer” (cf. https://sproget.dk/raad-og-regler/typiske-problemer/et-eller-flere-ord/et-eller-flere-ord/#3dcc7ef8-462b-4015-979b-a35239d24c31).

Reply:
Fixed!